cheatsheets/Clickjacking_Defense_Cheat_Sheet.md

```diff
@@ -1,9 +1,10 @@
-# Clickjacking Defense Cheat Sheet
+# Clickjacking and Double Clickjacking Defense Cheat Sheet
 
 ## Introduction
 
-This cheat sheet is intended to provide guidance for developers on how to defend against [Clickjacking](https://owasp.org/www-community/attacks/Clickjacking), also known as UI redress attacks.
+This cheat sheet is intended to provide guidance for developers on how to defend against [Clickjacking](https://owasp.org/www-community/attacks/Clickjacking), also known as UI redress attacks and Double Click Jacking. 
```

Comment:
Should be "Double Clickjacking", not "Double Click Jacking".

Also, let's give credit where credit is due, shall we and link the "Double Clickjackin" to Yibelo's blog post at https://www.paulosyibelo.com/2024/12/doubleclickjacking-what.html?m=1.